test/rating.js

```diff
@@ -16,40 +16,13 @@ describe('A track in Google Music', function () {
   });
   browserUtils.execute(function playViaApi () {
     window.googleMusic.playback.playPause();
+    window.googleMusic.playback.toggleRepeat();
```

Comment:
Their already appears to be `setRepeat` functionality hidden inside the `toggleRepeat` method, I'll just hook into that